resources/CICD/GithubActions/deploy_Firebase.yml

```diff
@@ -0,0 +1,79 @@
+name: deploy_Firebase
```

Comment:
Please update the name of the file and the value of the name here.
File name: `deploy_firebase.yml`
The value of the name: `name: Deploy Build To Firebase`